scripts/travis-pr-title-lint

```diff
@@ -138,13 +143,25 @@ function lintMessage(message) {
     throw new Error('Unable to lint an empty message.');
   }
 
-  const res = shelljs.exec(`echo ${message} | commitlint`, {silent: true});
+  try {
+    return execSync('commitlint', {
+      input: message,
+      windowsHide: true,
+      encoding: 'utf-8',
+    }).trim();
+  } catch (e) {
+    // execSync failure or timeouts.
+    if (e.error) {
+      throw e.error;
+    }
 
-  if (res.code !== 0) {
-    throw res.stdout;
+    // commitlint non-zero exit
+    if (e.status) {
+      // stderr by default will be output to the parent process' stderr (See
```

Comment:
How does this comment relate to the `throw stdout`? Should I read this as if you had ended the comment with `// so just throw stdout` ?

Reply:
yep,it is, let me add this part to the inline comment ;-)